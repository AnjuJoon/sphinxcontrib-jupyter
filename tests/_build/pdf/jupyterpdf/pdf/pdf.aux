\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand*\HyPL@Entry[1]{}
\HyPL@Entry{0<</S/D>>}
\providecommand \oddpage@label [2]{}
\@writefile{toc}{\contentsline {section}{\numberline {1}This is a test file for pdfs (and a Generalization)}{1}{section.1}}
\newlabel{this-is-a-test-file-for-pdfs-and-a-generalization}{{1}{1}{This is a test file for pdfs (and a Generalization)}{section.1}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.1}Contents}{1}{subsection.1.1}}
\newlabel{contents}{{1.1}{1}{Contents}{subsection.1.1}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.2}Notation}{1}{subsection.1.2}}
\newlabel{notation}{{1.2}{1}{Notation}{subsection.1.2}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.3}Model Ingredients and Assumptions (test case for brackets)}{2}{subsection.1.3}}
\newlabel{model-ingredients-and-assumptions-test-case-for-brackets}{{1.3}{2}{Model Ingredients and Assumptions (test case for brackets)}{subsection.1.3}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.4}Dynamic Interpretation}{2}{subsection.1.4}}
\newlabel{dynamic-interpretation}{{1.4}{2}{Dynamic Interpretation}{subsection.1.4}{}}
\newlabel{equation-this-equation-abc}{{{1}}{3}{Dynamic Interpretation}{AMS.1}{}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {1.4.1}Connection with Linear Programming (LP)}{3}{subsubsection.1.4.1}}
\newlabel{connection-with-linear-programming-lp}{{1.4.1}{3}{Connection with Linear Programming (LP)}{subsubsection.1.4.1}{}}
\newlabel{equation-this-eq2-xyz}{{{2}}{3}{Connection with Linear Programming (LP)}{AMS.2}{}}
\citation{test1458}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {1.4.2}Testing Citations}{4}{subsubsection.1.4.2}}
\newlabel{testing-citations}{{1.4.2}{4}{Testing Citations}{subsubsection.1.4.2}{}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {1.4.3}Testing Inline Math}{4}{subsubsection.1.4.3}}
\newlabel{testing-inline-math}{{1.4.3}{4}{Testing Inline Math}{subsubsection.1.4.3}{}}
