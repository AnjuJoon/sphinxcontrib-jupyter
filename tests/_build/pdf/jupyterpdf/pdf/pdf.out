\BOOKMARK [1][-]{section.1}{This\040is\040a\040test\040file\040for\040pdfs\040\(and\040a\040Generalization\)}{}% 1
\BOOKMARK [2][-]{subsection.1.1}{Contents}{section.1}% 2
\BOOKMARK [2][-]{subsection.1.2}{Notation}{section.1}% 3
\BOOKMARK [2][-]{subsection.1.3}{Model\040Ingredients\040and\040Assumptions\040\(test\040case\040for\040brackets\)}{section.1}% 4
\BOOKMARK [2][-]{subsection.1.4}{Dynamic\040Interpretation}{section.1}% 5
\BOOKMARK [3][-]{subsubsection.1.4.1}{Connection\040with\040Linear\040Programming\040\(LP\)}{subsection.1.4}% 6
\BOOKMARK [3][-]{subsubsection.1.4.2}{Testing\040Citations}{subsection.1.4}% 7
\BOOKMARK [3][-]{subsubsection.1.4.3}{Testing\040Inline\040Math}{subsection.1.4}% 8
